operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -505,6 +504,18 @@ default boolean useSSAToPatchPrimaryResource() {
     return true;
   }
 
+  /**
+   * Setting this to {@code true}, controllers will use SSA for adding finalizers.
```

Comment:
It might help to document why it might be a good idea to set this to false, in particular point to the Kubernetes issue.